src/snowflake/snowpark/column.py

```diff
@@ -605,7 +605,7 @@ def __str__(self):
         return f"Column[{self._expression}]"
 
     def __repr__(self):
-        return f"Column({self._expression})"
+        return f"Column({self._expression})"  # pragma: no cover
```

Comment:
Why is `__repr__` different from `__str__`? Also this might be testable?

Reply:
[`__repr__` vs `__str__`](https://stackoverflow.com/questions/1436703/what-is-the-difference-between-str-and-repr)
`assert eval(repr(x)) == x` does not apply for this case, so I chose to skip.

Reply:
Makes sense. I thought using parenthesis is not too bad, but anyways this is okay :)

Reply:
Makes sense. The problem is `Expression` doesn't override `__repr__`.